cmd/clusterctl/clusterdeployer/clusterclient/clusterclient.go

```diff
@@ -217,6 +219,17 @@ func (c *client) DeleteNamespace(namespaceName string) error {
 // NewFromDefaultSearchPath creates and returns a Client.  The kubeconfigFile argument is expected to be the path to a
 // valid kubeconfig file.
 func NewFromDefaultSearchPath(kubeconfigFile string, overrides tcmd.ConfigOverrides) (*client, error) { //nolint
+	var c ctrlclient.Client
+	if err := util.PollImmediate(retryAcquireClient, timeoutAcquireClient, func() (_ bool, err error) {
+		c, err = clientcmd.NewControllerRuntimeClient(kubeconfigFile, overrides)
+		if err != nil {
+			return false, err
```

Comment:
Should probably output something (even if just at a higher verbosity level) to avoid the user believing the process may be hung.

Reply:
Done! Added a log line with V = 2